terraform/api_gateway.tf.json.template.py

```diff
@@ -91,43 +91,21 @@ def for_domain(cls, domain):
 api_gateway_log_format = {
     'accountId': '$context.accountId',
     'apiId': '$context.apiId',
-    'authorizer_claims_property': '$context.authorizer.claims.property',
-    'authorizer_error': '$context.authorizer.error',
-    'authorizer_principalId': '$context.authorizer.principalId',
-    'authorizer_property': '$context.authorizer.property',
     'awsEndpointRequestId': '$context.awsEndpointRequestId',
-    'awsEndpointRequestId2': '$context.awsEndpointRequestId2',
-    'customDomain_basePathMatched': '$context.customDomain.basePathMatched',
-    'dataProcessed': '$context.dataProcessed',
     'domainName': '$context.domainName',
     'domainPrefix': '$context.domainPrefix',
     'error_message': '$context.error.message',
     'error_responseType': '$context.error.responseType',
     'extendedRequestId': '$context.extendedRequestId',
     'httpMethod': '$context.httpMethod',
-    'identity_accountId': '$context.identity.accountId',
-    'identity_caller': '$context.identity.caller',
-    'identity_cognitoAuthenticationProvider': '$context.identity.cognitoAuthenticationProvider',
-    'identity_cognitoAuthenticationType': '$context.identity.cognitoAuthenticationType',
-    'identity_cognitoIdentityId': '$context.identity.cognitoIdentityId',
-    'identity_cognitoIdentityPoolId': '$context.identity.cognitoIdentityPoolId',
-    'identity_principalOrgId': '$context.identity.principalOrgId',
-    'identity_clientCert_clientCertPem': '$context.identity.clientCert.clientCertPem',
-    'identity_clientCert_subjectDN': '$context.identity.clientCert.subjectDN',
-    'identity_clientCert_issuerDN': '$context.identity.clientCert.issuerDN',
-    'identity_clientCert_serialNumber': '$context.identity.clientCert.serialNumber',
-    'identity_clientCert_validity_notBefore': '$context.identity.clientCert.validity.notBefore',
     'identity_clientCert_validity_notAfter': '$context.identity.clientCert.validity.notAfter',
```

Comment:
According to [the instructions on the ticket](https://github.com/DataBiosphere/azul/issues/6399#issuecomment-2313056103), this field should be removed as well
```suggestion
```